### ISSUE TITLE
Notice: Block type is already registered running oikwp oik-shortcodes.php 

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-shortcodes#87]
When running oik-shortcodes.php to parse Jetpack I got the following messages.
What's the cause of this problem?
```
C:\apache\htdocs\oikcom\wp-content\plugins\oik-shortcodes>php C:\apache\htdocs\wordpress\wp-content\plugins\oik-batch\oik-wp.php oik-shortcodes.php jetpack url=jetpack.wp.a2z
oik-wp running WordPress 6.0.2
C:\apache\htdocs\oikcom\wp-content\plugins\oik-shortcodes
cli
Shifting argv
Script required once: oik-shortcodes.php
Running creatapi2 directly
Max execution time is: 0

Notice: Function WP_Block_Type_Registry::register was called <strong>incorrectly</strong>. Block type "oiksp/squeeze" is already registered. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 5.0.0.) in C:\apache\htdocs\oikcom\wp-includes\functions.php on line 5831
Doing a plugin: jetpack
Continue? ( y=yes/n=no/s=skip/g=go/q=quit )
```

**Comment by bobbingwide** [comment]
### Explanation
This is caused by the action `oik_loaded` being invoked more than once.

### Solution
In `oiksc_preloader()` call `did_action()` before `do_action()`

**Comment by bobbingwide** [comment]
Released in v1.41.2
